app/controllers/v0/messages_controller.rb

```diff
@@ -26,8 +26,12 @@ def show
     def create
       message = Message.new(message_params)
       raise Common::Exceptions::ValidationErrors, message unless message.valid?
-
-      response = client.post_create_message(message_params)
+      response =
+        if request.content_type == 'multipart/form-data'
```

Comment:
could this check for the presence of `params[:file]` rather than the content type?